Library/Homebrew/cmd/desc.rb

```diff
@@ -1,6 +1,12 @@
 #:  * `desc` <formula>:
 #:    Display <formula>'s name and one-line description.
 #:
+#:  * `desc` -a:
+#:    Display name and one-line description for all formulae.
+#:
+#:  * `desc` -i:
```

Comment:
If this remains part of desc it should be `--installed`. I think it may make more sense as `brew list --desc`, though.